modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing-vsphere.adoc [UPI]
+//* installing-vsphere-network-customizations.adoc [UPI]
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere region and zone enablement
+
+You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+[IMPORTANT]
+====
+The VMware vSphere region and zone enablement feature is only available with a newly installed cluster, because this feature requires the vSphere Container Storage Interface (CSI) driver as the default storage driver in the cluster.
+
+A cluster that was upgraded from a previous release defaults to using the in-tree vSphere driver, so you must enable CSI automatic migration for the cluster. You can then configure multiple regions and zones for the upgraded cluster. 
+====
+
+You must create an installation configuration file and then modify for the purposes of region and zone enablement, because deploying a cluster to a single vSphere datacenter is the default installation configuration.
+
+The default `install-config.yaml` file includes `vcenters` and `failureDomains` fields, where you can specify multiple vSphere datacenters and clusters for your {product-title} cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter.
+
+The following list describes terms associated with defining zones and regions for your cluster:
+
+* Failure domain: Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+* Region: Specifies a vCenter datacenter. You define a region by using a tag from the  `openshift-region` tag category. 
+* Zone: Specifies a vCenter cluster. You define a zone by using a tag from the `openshift-zone` tag category. 
```

Comment:
This should be a separate reference module and formatted as a definition list. See https://github.com/openshift/openshift-docs/blob/main/modules/install-openshift-common-terms.adoc for an example